Packs/Code42/Integrations/Code42/Code42_test.py

```diff
@@ -1205,19 +1215,23 @@ def test_client_lazily_inits_sdk(mocker):
     assert client._sdk is not None
 
 
-def test_client_when_no_alert_found_raises_exception(code42_sdk_mock):
-    code42_sdk_mock.alerts.get_details.return_value = (
-        """{'type$': 'ALERT_DETAILS_RESPONSE', 'alerts': []}"""
+def test_client_when_no_alert_found_raises_Code42AlertNotFoundError(mocker, code42_sdk_mock):
+    response_json = """{"alerts": []}"""
+    code42_sdk_mock.alerts.get_details.return_value = create_mock_code42_sdk_response(
+        mocker, response_json
     )
     client = create_client(code42_sdk_mock)
-    with pytest.raises(Exception):
+    with pytest.raises(Code42AlertNotFoundError):
         client.get_alert_details("mock-id")
 
 
-def test_client_when_no_user_found_raises_exception(code42_sdk_mock):
-    code42_sdk_mock.users.get_by_username.return_value = """{'totalCount': 0, 'users': []}"""
+def test_client_when_no_user_found_raises_Code42UserNotFoundError(mocker, code42_sdk_mock):
```

Comment:
something is wrong here, I corrected this in my legal hold PR but this is still in the diff, is this branch caught up with master?

Reply:
I think this may be creating duplicate tests (this branch has the test named differently)

Reply:
nope, looking at the file, my tests are just completely gone from this branch

Reply:
https://github.com/code42/content/blob/6d3d2ada3b385924031d03bc60d4144d27c4618d/Packs/Code42/Integrations/Code42/Code42_test.py#L1317 is what I'm looking for